relay-server/src/services/store.rs

```diff
@@ -511,6 +493,37 @@ impl StoreService {
             .client
             .send_message(topic, organization_id, &message)?;
 
+        match &message {
+            KafkaMessage::Span(span) => {
+                let is_segment = span.is_segment;
+                let has_parent = span.parent_span_id.is_some();
+                let platform = VALID_PLATFORMS.iter().find(|p| *p == &span.platform);
+
+                metric!(
+                    counter(RelayCounters::ProcessingMessageProduced) += 1,
+                    event_type = message.variant(),
+                    platform = platform.unwrap_or(&""),
+                    is_segment = bool_to_str(is_segment),
+                    has_parent = bool_to_str(has_parent),
+                );
+            }
+            KafkaMessage::ReplayRecordingNotChunked(replay) => {
+                let has_video = replay.replay_video.is_some();
+
+                metric!(
+                    counter(RelayCounters::ProcessingMessageProduced) += 1,
+                    event_type = message.variant(),
+                    has_video = bool_to_str(has_video),
```

Comment:
Is this new tag documented on `RelayCounters`? We should also let @cmanallen know that this is now reported in a different way.

Reply:
Before there were two different event types, it's now the same event type but using a tag. I'll shoot him a message on Slack 👍